Dockerfile

```diff
@@ -25,7 +25,16 @@ RUN echo "AUTOSTART=none" > /etc/default/openvpn \
 COPY package.json package-lock.json /usr/src/app/
 RUN npm install --unsafe-perm --production && npm cache clean --force 2>/dev/null
 COPY . /usr/src/app
-RUN npm run check
 
 COPY config/services /etc/systemd/system
-RUN systemctl enable open-balena-vpn.service open-balena-connect-proxy.service
+RUN systemctl enable open-balena-vpn-api.service open-balena-connect-proxy.service
+
+# build test image
+FROM base as test
+RUN npm install && npm cache clean --force 2>/dev/null
```

Comment:
This should be
```suggestion
RUN npm ci && npm cache clean --force 2>/dev/null

Reply:
Why does it need to be a clean install? It is just adding the development packages so the unit tests can be run.